selfdrive/car/toyota/interface.py

```diff
@@ -90,7 +90,8 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
       stop_and_go = False
       ret.safetyParam = 88
       ret.wheelbase = 2.70
-      ret.steerRatio = 18.27
+      ret.steerRatio = 17.43
```

Comment:
Can we put the steerRatio in a different PR? Doesn't have to do with longitudinal.

Reply:
You got it!